src/models/user.js

```diff
@@ -5,14 +5,17 @@ const passportLocalMongoose = require("passport-local-mongoose");
 const UserSchema = new Schema({
   user_type: {
     type: String,
-    enum: ["student", "instructor"], //Only allows users to be "students" or "instructors"
+    enum: ["student", "instructor"], // Only allows users to be "students" or "instructors"
     required: true,
   },
+  classes: [
```

Comment:
To improve robustness:
- Ensure validation checks are added to confirm that all referenced `Class` documents exist.
- Consider adding a mechanism to prevent duplicate class entries in this array.